Description: Fix FTBFS with glibc 2.23.
 Fixed bug in setting and reporting n_empty using NaN or proxy
Author: Paul Wessel
Origin: https://gmt.soest.hawaii.edu/projects/gmt/repository/revisions/15261
Bug: https://gmt.soest.hawaii.edu/issues/877
Bug-Debian: https://bugs.debian.org/818807

--- a/src/xyz2grd.c
+++ b/src/xyz2grd.c
@@ -749,7 +749,7 @@ int GMT_xyz2grd (void *V_API, int mode,
 		if (GMT_is_verbose (GMT, GMT_MSG_VERBOSE)) {
 			char line[GMT_BUFSIZ], e_value[GMT_LEN32];
 			sprintf (line, "%s\n", GMT->current.setting.format_float_out);
-			(GMT_is_dnan (GMT->common.d.active[GMT_IN])) ? sprintf (e_value, "NaN") : sprintf (e_value, GMT->current.setting.format_float_out, GMT->common.d.nan_proxy[GMT_IN]);
+			(GMT->common.d.active[GMT_IN]) ? sprintf (e_value, GMT->current.setting.format_float_out, GMT->common.d.nan_proxy[GMT_IN]) : sprintf (e_value, "NaN");
 			GMT_Report (API, GMT_MSG_VERBOSE, "Data records read: %" PRIu64 "  used: %" PRIu64 "  nodes filled: %" PRIu64 " nodes empty: %" PRIu64 " [set to %s]\n",
 				n_read, n_used, n_filled, n_empty, e_value);
 			if (n_bad) GMT_Report (API, GMT_MSG_VERBOSE, "%" PRIu64 " records unreadable\n", n_bad);
